src/main/resources/application.properties

```diff
@@ -10,6 +10,7 @@ spring.resources.cache.cachecontrol.must-revalidate=true
 spring.resources.static-locations=classpath:/api/
 db.migrate.dropAllOnStart=false
 db.migrate.updateAllOnStart=true
+db.migrate.dataProjectNoDropAll=broad-jade-dev-data,terra-datarepo-alpha-data
```

Comment:
we want staging too.  It's a little strange because these are also environment-specific but I get why you did it

Reply:
That's a good point. It's definitely protecting against an edge case where we're running stuff locally and point to another environment. But, hopefully we can get rid of all of this when we remove the "drop all" option from the code altogether.